database/seeders/modules/laravel-basics.php

```diff
@@ -0,0 +1,295 @@
+                'url' => 'https://laracasts.com/series/laravel-6-from-scratch/episodes/37',
+                'type' => Resource::VIDEO_TYPE,
+            ],
+            [
+                'name' => 'Laravel Docs (Validation)',
+                'url' => 'https://laravel.com/docs/6.x/validation',
+                'type' => Resource::ARTICLE_TYPE,
+            ],
+            [
+                'name' => 'Laravel Docs (Mail)',
+                'url' => 'https://laravel.com/docs/6.x/mail',
+                'type' => Resource::ARTICLE_TYPE,
+            ],
+            [
+                'name' => 'Laravel Docs (Notifications)',
+                'url' => 'https://laravel.com/docs/6.x/notifications',
+                'type' => Resource::ARTICLE_TYPE,
+            ],
+            [
+                'name' => 'Laravel Docs (Container)',
+                'url' => 'https://laravel.com/docs/6.x/container',
+                'type' => Resource::ARTICLE_TYPE,
+            ],
+            [
+                'name' => 'The Laravel Podcast - The Ethos of Laravel, with Taylor Otwell',
+                'url' => 'https://laravelpodcast.com/episodes/the-ethos-of-laravel',
+                'type' => Resource::AUDIO_TYPE,
+            ],
+            [
+                'name' => 'The Laravel Podcast - Learning & Keeping Up To Date, with Eric Barnes',
+                'url' => 'https://laravelpodcast.com/episodes/learning-keeping-up-to-date-with-eric-barnes',
+                'type' => Resource::AUDIO_TYPE,
+            ],
+            [
+                'name' => 'The Laravel Podcast - Routing and Blade, with Caleb Porzio',
+                'url' => 'https://laravelpodcast.com/episodes/routing-and-blade-with-caleb-porzio',
+                'type' => Resource::AUDIO_TYPE,
+            ],
+            [
+                'name' => 'The Laravel Podcast - Eloquent and the Query Builder, with Jonathan Reinink',
+                'url' => 'https://laravelpodcast.com/episodes/eloquent-with-jonathan-reinink',
+                'type' => Resource::AUDIO_TYPE,
+            ],
+            [
+                'name' => 'The Laravel Podcast - Migrations, Factories, and Seeding, with John Bonaccorsi',
+                'url' => 'https://laravelpodcast.com/episodes/migrations-factories-and-seeding-with-john-bonaccorsi',
+                'type' => Resource::AUDIO_TYPE,
+            ],
+            [
+                'name' => 'The Laravel Podcast - Authorization and Authentication, with Joseph Silber',
+                'url' => 'https://laravelpodcast.com/episodes/authorization-and-authentication-with-joseph-silber',
+                'type' => Resource::AUDIO_TYPE,
+            ],
+            [
+                'name' => 'The Laravel Podcast - Presets & Jetstream, with Michael Dyrynda',
+                'url' => 'https://laravelpodcast.com/episodes/presets-jetstream-with-michael-dyrynda',
+                'type' => Resource::AUDIO_TYPE,
+            ],
```

Comment:
Proposal: This should be a part of a new intermediate or advanced module surrounding Authentication/Authorization
```suggestion
```